src/frontends/tensorflow_common/include/tf_framework_node.hpp

```diff
@@ -15,6 +15,7 @@ namespace tensorflow {
 
 class FrameworkNode : public ov::op::util::FrameworkNode {
 public:
+    static constexpr const char* failed_conversion_key = "tensorflow::FrameworkNode::failed_conversion_key";
```

Comment:
Not a blocker for the PR but just for the records, this `failed_conversion_key` looks bulky in comparison to a dedicate field in the class.

Reply:
Agreed to proceed with the current version because this is not clear how to implement this better.